samples/tab-ui-templates/ts/.gitignore

```diff
@@ -84,3 +84,25 @@ yarn-error.log*
 npm-debug.log*
```

Comment:
For tab-ui-templates, you could refer to latest https://github.com/formulahendry/Microsoft-Teams-Samples/tree/junhan/v3/samples/tab-ui-templates/ts

Reply:
changed